### ISSUE TITLE
Error: tsx must be loaded with --import instead of --loader

### ISSUE THREAD

**Issue by dangnhdev** [Lordfirespeed/idea-run-typescript#3]
```
node:internal/process/esm_loader:40
      internalBinding('errors').triggerUncaughtException(
                                ^
Error: tsx must be loaded with --import instead of --loader
The --loader flag was deprecated in Node v20.6.0
    at V (file:///project/node_modules/tsx/dist/esm/index.mjs:1:1956)
    at Hooks.addCustomLoader (node:internal/modules/esm/hooks:178:24)
    at Hooks.register (node:internal/modules/esm/hooks:144:16)
    at async initializeHooks (node:internal/modules/esm/utils:184:5)
    at async customizedModuleWorker (node:internal/modules/esm/worker:86:24)

Node.js v20.9.0
```
I've just tried this new plugin and got this error. The file run without error with old plugin.

**Comment by Lordfirespeed** [comment]
I think this is partially due to changes in Node.js and TSX rather than this plugin, but I should be able to implement a fix!

**Comment by Lordfirespeed** [comment]
After some investigation ...
- `--import` is available in all versions of Node `>=18.18.0`
  - (unimportant) it is preferable in Node `>=18.19.0, <19` and Node `>=20.6.0`, which include https://github.com/nodejs/node/commit/a40a6c890afe0d1d0ca78db015146178c25af079, https://github.com/nodejs/node/commit/6ad8318373664e5d9c4f4133c9393c59c720f6a9 and https://github.com/nodejs/node/commit/a2fc4a383efe8b607bacc3256c834a4ba78c66d2
- For versions of Node `<18.18.0`, `--import` is not available so `--loader` (which is an alias for `--experimental-loader`) must be used 

So, to resolve this issue - 
- [ ] When Node version `>=18.18.0`, use `--import` over `--require` in all possible scenarios
  - [ ] check for compatability with `ts-node`
  - [x] check for compatability with `esno` and `tsx`
- [ ] When Node version `<18.18.0`, use `--require` in all possible scenarios

**Comment by nullice** [comment]
Perhaps you can provide an option directly

**Comment by Lordfirespeed** [comment]
> Perhaps you can provide an option directly

Yes, that's my intention. I haven't got round to it. Feel free to contribute :)
